i18n/detect.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+Function to check whether or not the English strings are up to date
+"""
+from __future__ import print_function
+from subprocess import CalledProcessError
+
+from i18n import Runner
+from i18n.execute import execute
+from i18n.extract import Extract
+
+
+class Detect(Runner):
+    """
+    Class used to check if the English source strings are up to date
+    """
+    def run(self, args):
+        """
+        Main entry point of script
+        """
+        Extract().run(args)  # Run extract command
```

Comment:
Should this command call extract itself, or can we let the caller of "detect" run "extract" first?